pulsar-rabbitmq-gw/src/main/resources/META-INF/services/pulsar-proxy-protocol-handler.yml

```diff
@@ -1,3 +1,3 @@
 name: rabbitmq
-description: RabbitMQ Proxy Protocol Handler
-handlerClass: com.datastax.oss.pulsar.rabbitmqgw.GatewayProxyProtocolHandler
\ No newline at end of file
+description: RabbitMQ Proxy Extension
+extensionClass: com.datastax.oss.pulsar.rabbitmqgw.GatewayProxyProtocolHandler
```

Comment:
I believe that you have to rename this file. it is no more pulsar-proxy-protocol-handler.
is it "pulsar-proxy-extension.yaml" ?

Reply:
renamed